### PR TITLE
fix: observer pull not updating last txn number that pulled

### LINKED ISSUE #55: observer pull not updating last txn number that pulled

### PR DESCRIPTION
**Pull request by fqutishat**
closes #55

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/observer/observer.go [hunk 1]
```diff
@@ -35,7 +35,8 @@ func (l *ledger) RegisterForSidetreeTxn() <-chan []sidetreeobserver.SidetreeTxn
 				var sidetreeTxn *sidetreeobserver.SidetreeTxn
 				moreTransactions, sidetreeTxn = l.blockChainClient.Read(sinceTransactionNumber)
 				if sidetreeTxn != nil {
-					logger.Debugf("found sidetree txn %d in ledger", sidetreeTxn.TransactionNumber)
+					sinceTransactionNumber = int(sidetreeTxn.TransactionNumber)
```
Comment (sandrask) [inline pkg/observer/observer.go]: Do we need a test for this?

File: pkg/observer/observer.go [hunk 2]
```diff
@@ -35,7 +35,8 @@ func (l *ledger) RegisterForSidetreeTxn() <-chan []sidetreeobserver.SidetreeTxn
...
+					logger.Infof("found sidetree txn %d in ledger", sidetreeTxn.TransactionNumber)
```
Comment (troyronda) [inline pkg/observer/observer.go]: Shouldn't this remain Debugf?

File: pkg/observer/observer.go (hunk 1 above)
Comment (fqutishat) [reply pkg/observer/observer.go]: done

File: pkg/observer/observer.go (hunk 2 above)
Comment (fqutishat) [reply pkg/observer/observer.go]: done

**Pull request merged**